arangod/IResearch/IResearchFilterFactory.cpp

```diff
@@ -1044,10 +1044,53 @@ Result fromRange(irs::boolean_filter* filter, QueryContext const& /*ctx*/,
 
 std::pair<Result, aql::AstNodeType> buildBinaryArrayComparisonPreFilter(
     irs::boolean_filter*& filter, aql::AstNodeType arrayComparison,
-    const aql::AstNode* quantifierNode, size_t arraySize) {
+    const aql::AstNode* quantifierNode, size_t arraySize,
+    QueryContext const& ctx) {
   TRI_ASSERT(quantifierNode);
   auto quantifierType =
       static_cast<aql::Quantifier::Type>(quantifierNode->getIntValue(true));
+
+  int64_t atLeastCount{1};
```

Comment:
Why is 1 the default?

Reply:
To use with  kAny. As kAny is   atLeast(1)